### ISSUE TITLE
Merge 'GO:0019836 hemolysis by symbiont of host erythrocytes' into parent? 

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#15553]
Hello, 

GO:0044179 'hemolysis in other organism' is defined as 'The cytolytic destruction of red blood cells, with the release of intracellular hemoglobin, in one organism by another.'

I don't see the the difference with 'GO:0019836 hemolysis by symbiont of host erythrocytes' (The cytolytic destruction of red blood cells, with the release of intracellular hemoglobin, in the host organism by a symbiont. The host is defined as the larger of the organisms involved in a symbiotic interaction.) 

(nor with GO:0052331 hemolysis in other organism involved in symbiotic interaction) 

I propose merging those 3 terms into the most specific term, 'GO:0019836 hemolysis by symbiont of host erythrocytes'. 

@suzialeksander 
@fjungo 
@pgarmiri 

You have commented on previous related tickets or annotated to these terms - let me know of you have any comments on this. 

Thanks, Pascale

**Comment by pgaudet** [comment]
@FJungo mentions tha we need to take into account venoms - cause hemolysis without being a symbiont.

**Comment by pgarmiri** [comment]
I agree with that. Most of the case I had to review in a previous ticket were or that type. (peptides produced by frogs/fish that have hemolytic activity against human/rabbit red blood cells. ) https://github.com/geneontology/go-annotation/issues/1379

**Comment by pgaudet** [comment]
Thanks ! That wasn't clear at all from the term definitions, but it seems a useful distinction. 

Should we have something like: 
- hemolysis by symbiont
- hemolysis by secreted/injected(?) compound

Thanks, Pascale

**Comment by pgarmiri** [comment]
That sounds reasonable to me. Perhaps, at the second term an example could be added in the end , for example '...such as venom'  pointing to a relevant publication. I could easily find one if you would like.
Thanks,
Penelope

**Comment by suzialeksander** [comment]
I don't know that the distinction between 'by symbiont' and 'by secreted compound' always exists.

Perhaps @sandyl27 can give input from the bacterial/viral world?

**Comment by sandyl27** [comment]
@jimhu-tamu

**Comment by jimhu-tamu** [comment]
I'm afraid there are issues with both "secreted" and "symbiont" if we want to be consistent in how toxins are handled. I'm not sure whether hemolysins are represented in what follows, but the examples for microbial toxins in general should be considered.

As noted above, the toxins from frogs and snake venoms can include hemolytic substances. Some are proteins and others may not be. Fire ant venom has hemolytic components that appear to be small molecules.  In some cases this runs into the question of whether predation (or its avoidance) is a form of symbiosis. While some sources consider predation to be symbiosis, I don't think that's the current usage in GO, where the paradigm is a host-symbiont relationship.

It's not clear that all hemolytic toxins are secreted. The Shiga Toxin in E. coli food poisoning outbreaks is made by a prophage. It is thought that the toxin is released when an induced phage kills the host bacterium, which would not be thought of as secretion. Similarly, there are toxins that are found inside spores that may be released during germination.  I don't think it's a hemolysin, but the toxin in Bacillus thuringiensis that's used in GM crops forms intracellular crystalline inclusions. I think the cells have to die to release the toxin.
